clients/ui/frontend/src/app/hooks/modelCatalog/useCatalogSources.ts

```diff
@@ -11,8 +11,10 @@ export const useCatalogSources = (
       if (!apiState.apiAvailable) {
         return Promise.reject(new Error('API not yet available'));
       }
-
-      return apiState.api.getListSources(opts);
+      return apiState.api.getListSources(opts).then((data) => ({
+        ...data,
+        items: data.items ?? [],
+      }));
```

Comment:
This isn't strictly necessary since the type change means we defend against missing items elsewhere, but it doesn't hurt.